### ISSUE TITLE
$(AWK) is undefined

### ISSUE THREAD

**Issue by kmihaylov** [SmingHub/Sming#1997]
Good morning,

This shouldn't be a thing since Sming compiles successfully despite the error, but when I do ```make dist-clean``` I'm getting this warning:
```
$ make dist-clean 

general: Invoking 'dist-clean' for Esp8266 (debug) architecture
Cleaning all submodules...
Cleaning all samples...
/bin/bash: BEGIN { big=0; for(i = 1; i < ARGC; ++i) if(ARGV[i] > 1024*1024) big=1; print big; }: command not found
../samples/HttpServer_FirmwareUpload/component.mk:10: warning: overriding recipe for target 'web-pack'
../samples/HttpServer_ConfigNetwork/component.mk:5: warning: ignoring old recipe for target 'web-pack'
../samples/HttpServer_FirmwareUpload/component.mk:13: warning: overriding recipe for target 'web-upload'
../samples/HttpServer_ConfigNetwork/component.mk:9: warning: ignoring old recipe for target 'web-upload'
```

Setting a variable ```export AWK=/usr/bin/awk``` solves the problem.

**Comment by slaff** [comment]
@kmihaylov Did you pull the latest `develop` version? That should have been fixed.

**Comment by mikee47** [comment]
Try a git pull, this is fixed in latest commit 3a99771e

**Comment by kmihaylov** [comment]
Thanks! The warning is gone.
